api/v2/api.proto

```diff
@@ -14,6 +14,13 @@ message Client {
   bool public = 5;
   string name = 6;
   string logo_url = 7;
+  SamlInitiatedConfig saml_initiated = 8;
```

Comment:
Adding settings for a particular connector into generic structures is not good. Introducing a special handler for SAML is okay, but adding these options to ALL connectors could be better.